Includes/Rosetta/Enchants/Effects.hpp

```diff
@@ -30,6 +30,12 @@ class Effects
         return new Effect(GameTag::HEALTH, EffectOperator::ADD, n);
     }
 
+	//! Creates effect that increases health by \p n.
```

Comment:
You have to correct comments (not `health` but `durability`) and rearrange whitespaces.

Reply:
Also, you have to add comment for parameter and return type.

Reply:
The other functions in Effects.hpp don't have a comment on return type or paramter.

Reply:
Hmm... Other functions have to add comment for parameter and return type too. Can you add it?

Reply:
Ok. I'll try it.

Reply:
Thanks. 😄